contracts/0.8.9/WithdrawalQueue.sol

```diff
@@ -340,20 +339,57 @@ contract WithdrawalQueue is AccessControlEnumerable, WithdrawalQueueBase {
         _finalize(_lastRequestIdToFinalize, msg.value);
     }
 
+    /**
+     * @notice Update bunker mode state
+     * @dev should be called by oracle
+     *
+     * NB: timestamp should correspond to the previous oracle report
+     *
+     * @param _previousOracleReportTimestamp timestamp of the previous oracle report
+     * @param _isBunkerModeNow oracle report
+     */
+    function updateBunkerMode(
```

Comment:
Maybe the other way around, `updateBunkerMode(_isBunkerModeNow, _previousOracleReportTimestamp)`? This way it reads better imo.

Reply:
no problem, fixed